src/components/Highlights.tsx

```diff
@@ -0,0 +1,62 @@
+import { LINKS } from '@constants';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  crossPlatfom, freeOpen, since1979, trustedMilitary,
+} from '../assets/images';
+import { HighlightBlock } from './common';
+
+interface IProps { }
+
+export const Highlights: React.FC<IProps> = () => {
+  const { t } = useTranslation();
+  const highlights = [
+    {
+      index: 1,
```

Comment:
Do we need index here, can that automatically be determined from its position in the array?

Reply:
I am not even sure now why i added this one 
but I have fixed it and pushed